### PR TITLE
bzip2: unify in one recipe

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by SSE4 — APPROVED** [review]

File: recipes/bzip2/all/conanfile.py [hunk 1]
```diff
@@ -5,7 +5,6 @@
 
 class Bzip2Conan(ConanFile):
     name = "bzip2"
-    version = "1.0.8"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "http://www.bzip.org"
     license = "bzip2-1.0.8"
```
Comment (mathbunnyru) [inline recipes/bzip2/all/conanfile.py:10]:
```suggestion
    license = "bzip2"
```

I would prefer not to see some version here.

**Review by mathbunnyru — APPROVED** [review]

File: recipes/bzip2/all/conanfile.py (hunk 1 above)
Comment (uilianries) [reply recipes/bzip2/all/conanfile.py:10]: That's part of license name: https://spdx.org/licenses/bzip2-1.0.6.html

**Review by uilianries — APPROVED** [review]
LGTM

File: recipes/bzip2/all/conanfile.py (hunk 1 above)
Comment (mathbunnyru) [reply recipes/bzip2/all/conanfile.py:10]: But at the same time bzip2-1.0.8 license doesn't exist, if I'm right.

**Pull request merged**